src/org/usfirst/frc/team2175/subsystem/drivetrain/DrivetrainSubsystem.java

```diff
@@ -114,14 +134,8 @@ public void switchToPercentVbus() {
         rightMasterMotor.changeControlMode(TalonControlMode.PercentVbus);
     }
 
-    public void resetEncoders() {
-        leftMasterMotor.setEncPosition(0);
-        rightMasterMotor.setEncPosition(0);
-
-    }
-
     public double getCurrentEncPosition() {
-        return leftMasterMotor.getEncPosition();
+        return rightMasterMotor.getEncPosition();
```

Comment:
@NoahConnors This method should probably just be deleted, since we're not using encoder values from the Talons.